src/main/java/views/create_new_month/NewMonthC.java

```diff
@@ -0,0 +1,36 @@
+package views.create_new_month;
+
+import entities.SessionStorage;
+import use_cases.create_new_month.NewMonthIB;
+import use_cases.create_new_month.NewMonthID;
+import use_cases.create_new_month.NewMonthOD;
+
+/**
+ * The controller class for creating new MonthlyStorage. The view class
+ * calls this class to get the NewMonthOD object. This class generates the
+ * input data class and calls the input boundary interface to get the
+ * NewMonthOD object.
+ */
+public class NewMonthC {
+    final NewMonthIB inputBoundary;
+
+    /**
+     * Construct the controller class.
```

Comment:
To keep up with your previous naming conventions you should specify NewMonthC instead of controller.

Reply:
I've been using controller (and for other, input data, use case interactor, etc.) in full to make it more descriptive.

Reply:
Okay, sounds good!